### PR TITLE
admin add word answer

### PR DESCRIPTION
**Pull request by ChiThanh93**

### CODE REVIEW

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -11,4 +11,29 @@ def full_title page_title = ""
...
+  def link_to_remove_fields name, f
+    f.hidden_field(:_destroy) + link_to_function(name, "remove_fields(this)",
+      class: "btn fa fa-trash btn-danger")
+  end
+
+  def link_to_add_fields name, f, association
+    new_object = f.object.class.reflect_on_association(association).klass.new
+    fields = f.fields_for(association, new_object,
+      child_index: "correct_answer") do |builder|
+      render association.to_s.singularize + "_fields", f: builder
```
Comment (hathanhtamnd) [inline app/helpers/application_helper.rb]: indent

File: app/views/admin/words/_word.html.erb [hunk 2]
```diff
@@ -0,0 +1,24 @@
+<%= form_for [:admin, @word] do |f| %>
+  <%= render "shared/error_messages" , object: @word %>
+  <div class="form-group">
+  <%= f.select :category_id, options_for_select(@category.collect{|category| [category.name,
+   category.id]},), class: "form-control" %>
+  </div>
+
+  <div class="form-group">
+  <%= f.label :question %>
```
Comment (hathanhtamnd) [inline app/views/admin/words/_word.html.erb]: indent

File: app/views/admin/words/_word.html.erb [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  <%= f.text_field :content, class: "form-control" %>
```
Comment (hathanhtamnd) [inline app/views/admin/words/_word.html.erb]: Indent

File: app/controllers/admin/words_controller.rb [hunk 4]
```diff
@@ -0,0 +1,27 @@
+class Admin::WordsController < ApplicationController
+  before_action :load_all_category, only: [:new, :create]
+
+  def new
+    @word = Word.new
```
Comment (thangtx88) [inline app/controllers/admin/words_controller.rb]: @word = @category.words.build

**Pull request merged**